### PR TITLE
Discount tunes in time remaining

### PR DESCRIPTION
**Pull request by dubslow**
and misc cleanups in the function

### CODE REVIEW

File: server/fishtest/util.py [hunk 1]
```diff
@@ -343,17 +343,18 @@ def is_active_sprt_ltc(run):
 
 
 def remaining_hours(run):
+    if "spsa" in run["args"]: # Tunes are frequently stopped short, and with "autoselect" may have too many games anyways
+        return 0
     r = run["results"]
+    N = sum(r["pentanomial"]) if "pentanomial" in r else r["wins"] + r["losses"] + r["draws"]
```
Comment (ppigazzini) [inline server/fishtest/util.py:349]: Check if this is the intent:
 - a run with "pentanomial" field has "wins", "losses", "draws" fields as well
 - `sum(r["pentanomial"])` is half the number of games played

File: server/fishtest/util.py [hunk 2]
```diff
@@ -343,17 +343,18 @@ def is_active_sprt_ltc(run):
 
 
 def remaining_hours(run):
+    if "spsa" in run["args"]: # Tunes are frequently stopped short, and with "autoselect" may have too many games anyways
```
Comment (ppigazzini) [inline server/fishtest/util.py:346]: What about adding a time scale factor for SPSA?

**Comment by MinetaS** [comment]
To be honest I don't agree with this one.
It's named "remaining hours" not "expected remaining hours" ... for fixed games tests, this metric should be consistent.

File: server/fishtest/util.py (hunk 1 above)
Comment (dubslow) [reply server/fishtest/util.py:349]: ah, i was borrowing from Mineta's code lol, and did not stop to think about it

**Comment by dubslow** [comment]
> It's named "remaining hours" not "expected remaining hours" ... for fixed games tests, this metric should be consistent.

Well fixed games tests have wildly differing behavior depending on if it's a tune or not. Nontunes finish their full count of games with high probability, whereas tunes fall well short of their full count with high probability.

However, as ppigazzini suggests, perhaps discounting tunes is better than ignoring them altogether.

**Comment by ppigazzini** [comment]
Closed after merging #1545

**Pull request closed without merging**